tabular/src/autogluon/tabular/models/tab_transformer/utils.py

```diff
@@ -0,0 +1,138 @@
+from . import TabTransformerEncoder
+from .TabTransformerEncoder import WontEncodeError, NullEnc
+from ...try_import import try_import_torch
+
+def augmentation(data, target, mask_prob=0.4, num_augs=1):
```

Comment:
nit: 2 newlines between functions and classes. 2 newlines between final `from ... import` and function

Reply:
Updating style. 1 space between functions, 2 spaces between a function and a class and 2 spaces after final import?

Reply:
2 space between functions, 2 spaces between classes, 2 spaces between function and a class, 2 spaces after final import, 1 space between methods (inside a class). Refer to existing Tabular code, it is consistent in this regard.

Reply:
This is based off the default PEP8 settings for PyCharm

Reply:
Just sent all the code through PEP-8 autoformatter. Should fix any lingering style issues, but may be lots of small formatting changes in this latest commit.